watertap/costing/unit_models/cstr.py

```diff
@@ -0,0 +1,45 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+from ..util import (
+    register_costing_parameter_block,
+    cost_by_flow_volume,
+)
+
+
+def build_cstr_cost_param_block(blk):
+    # Source: https://www.fwrj.com/articles/9812.pdf
+    blk.sizing_cost = pyo.Var(
+        initialize=0.34,
```

Comment:
This is the unit cost for the largest size shown in the range from the reference. Perhaps we should choose a value like the mean or median.

Reply:
The reason I chose the largest size is because the flowrate in BSM2 actually goes way beyond the ranges listed in this reference. The largest flow in the reference is 100,000 GPD (378.5 m3/day) whereas the BSM2 feed flowrate is 20648 m3/day. So I didn't think it made sense to choose a mean or median value. Although if we intend on using this CSTR for non-BSM2 purposes, then it would make more sense to use an average value.

Reply:
Fair enough!